saleor/plugins/webhook/utils.py

```diff
@@ -127,6 +132,96 @@ def parse_excluded_shipping_methods_response(
             logger.warning(f"Malformed ShippingMethod id was provided: {e}")
             continue
         excluded_methods.append(
-            ExcludedShippingMethod(id=method_id, reason=method_data.get("reason", ""))
+            {"id": method_id, "reason": method_data.get("reason", "")}
```

Comment:
Why cannot you use here `ExcludedShippingMethod` dataclass?

Reply:
`exclude_methods` will be stored in cache, so I don't want to have any structure which can be problematic to handle